### PR TITLE
Update Allow List for MTRW, en-* and PTT disable

### PR DESCRIPTION
**Pull request by NancyForero**
Adding IT admin doc related to:
Update Allow List for MTRW
Support for en-* 
how to disable PTT

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/cortana-in-teams.md [hunk 1]
```diff
@@ -65,6 +70,12 @@ This example shows updating the policy and enabling Cortana voice assistance wit
 PS C:\> Set-CsTeamsCortanaPolicy -Identity EmployeeCortanaPolicy -CortanaVoiceInvocationMode WakeWordPushToTalkUserOverride
 ```
 
+This example shows how to disable Cortana push button and wake word invocation.
+
+```PowerShell
+PS C:\> Set-CsTeamsCortanaPolicy -Identity MyCortanaPolicy -CortanaVoiceInvocationMode Disabled
+```
+
 At the time, of the initial release for Microsoft 365 Enterprise users in the US in English, the following are available functions:
```
Comment (JohanFreelancer9) [inline Teams/cortana-in-teams.md:79]:
```suggestion
At the time of the initial release for Microsoft 365 Enterprise users in the US in English, the following are available functions:
```

File: Teams/cortana-in-teams.md [hunk 2]
```diff
@@ -93,13 +104,18 @@ At the device level, you can configure Cortana to be used in two different ways.
 - by tapping on a microphone, which is called Cortana _Push to talk_
 - by saying "Hey, Cortana", which is called _Cortana Voice Activation_
 
-Cortana _Push to talk_ is enabled by default if your device is set up to use locale en-us. [Learn More.](/MicrosoftTeams/rooms/console#to-apply-your-desired-language)  Cortana icon will displace the _Present_ button under the _More…_ menu in your Teams Room console. To disable Cortana _Push to talk_ use PowerShell.[Learn More.](/powershell/module/skype/new-csteamscortanapolicy?view=skype-ps#example-1)
+#### Cortana Push to Talk
+
+- Cortana _Push to talk_ is enabled by default if your device is set up to use locale en-us. [Learn More.](https://docs.microsoft.com/en-us/MicrosoftTeams/rooms/console#to-apply-your-desired-language) Cortana icon will displace the _Present_ button under the _More…_ menu in your Teams Room console.
```
Comment (JohanFreelancer9) [inline Teams/cortana-in-teams.md:109]:
```suggestion
- Cortana _Push to talk_ is enabled by default if your device is set up to use locale en-us. [Learn More.](/MicrosoftTeams/rooms/console#to-apply-your-desired-language) The Cortana icon will displace the _Present_ button under the _More…_ menu in your Teams Room console.
```

File: Teams/cortana-in-teams.md [hunk 3]
```diff
@@ -93,13 +104,18 @@ At the device level, you can configure Cortana to be used in two different ways.
...
+  - To revert UI console to show the Present button, Cortana _Push to Talk_ will need to be disabled via Powershell commands. [Learn More.](https://docs.microsoft.com/en-us/powershell/module/skype/set-csteamscortanapolicy?view=skype-ps#example-1)
```
Comment (JohanFreelancer9) [inline Teams/cortana-in-teams.md:110]:
```suggestion
  - To revert the UI console to show the Present button, Cortana _Push to Talk_ will need to be disabled via Powershell commands. [Learn More.](/powershell/module/skype/set-csteamscortanapolicy?view=skype-ps#example-1)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @NancyForero, kindly commit the minor suggested changes if you agree with it. Many thanks.

**Pull request closed without merging**